bridge/client/app/_components/ktb-create-secret-form/ktb-create-secret-form.component.html

```diff
@@ -0,0 +1,50 @@
+<div fxFlexFill>
+  <div class="container" fxFlex fxLayout="column">
+    <div class="mb-3 settings-section" fxFlex fxLayout="column">
+      <h2>Create Secret</h2>
+      <p class="mt-0 mb-3">
+        Create a secret to store e.g. credentials for the Keptn API or other integrations.
+      </p>
+
+      <form [formGroup]="createSecretForm">
+        <div class="mb-3" fxLayout="row" fxLayoutAlign="none end" fxLayoutGap="15px">
+          <dt-form-field>
+            <dt-label>Name</dt-label>
+            <input [(ngModel)]="secret.name" formControlName="name" type="text" required dtInput placeholder="Secret name" />
+            <dt-error>Must not be empty</dt-error>
+          </dt-form-field>
+        </div>
+
+        <div class="mb-1" fxLayout="row" fxLayoutAlign="none end" fxLayoutGap="15px" *ngFor="let data of secret.data; let index = index;">
+          <dt-form-field>
+            <dt-label>Key</dt-label>
+            <input [(ngModel)]="data.key" [formControlName]="'key'+(index+1)" type="text" required dtInput placeholder="Key" />
```

Comment:
The key also has some limitations
`valid config key must consist of alphanumeric characters, '-', '_' or '.' (e.g. 'key.name',  or 'KEY_NAME',  or 'key-name', regex used for validation is '[-._a-zA-Z0-9]+')`